module1/homework/grayscaleImages/main.cpp

```diff
@@ -1,7 +1,8 @@
+#include "compression.hpp"
+
 #include <array>
 #include <forward_list>
-
-// TODO: include
+#include <cstdint>
```

Comment:
You didn't need to add cstdint here since it's in the header you have included.For the same reason you coulde remove array.